src/components/ui/Panel.jsx

```diff
@@ -104,8 +104,6 @@ export default class Panel extends React.Component {
   }
 
   holdResizer = event => {
-    event.preventDefault();
```

Comment:
Can you explain why this is needed?

Reply:
I restored the old code as we want to prevent mouse events to be raised when touch event has occured.